roentgen/continuum/emission.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+
+import astropy.units as u
+from astropy.constants.codata2018 import h, c, e, k_B
+
+__all__ = [
+    "thermal_bremsstrahlung",
+    "mewe_gaunt",
+    "xray_tube_spectrum",
+    "cutoff_wavelength"
+]
+
+
+def thermal_bremsstrahlung(wavelength, z, density, temperature):
+    """
+    This equation assumes that the density of ions is equal to the density of
+    electrons.
+    """
+    frequency = wavelength.to('Hz', equivalencies=u.spectral())
+    return z ** 2 * density ** 2 * np.sqrt(1 / (k_B * temperature)) * \
+           np.exp(- h * frequency / (k_B * temperature)) * mewe_gaunt(wavelength, temperature)
+
+
+def mewe_gaunt(wavelength, temperature):
+    """Based on Equation 3 in Mewe 1985."""
+    return 27.83 * (temperature.to('MK').value + 0.65) ** (-1.33) + 0.15 * wavelength.to('angstrom').value ** (0.34) * temperature.to('MK').value ** (0.422)
```

Comment:
I'd say its best to `use to_value()`
`return 27.83 * (temperature.to_value('MK')+ 0.65) ** (-1.33) + 0.15 * wavelength.to_value('angstrom') ** (0.34) * temperature.to_value('MK') ** (0.422)`

Reply:
also similarly require that wavelength and temperature are astropy quantities with the correct units (`quantity_input`)

Reply:
I believe this line still needs the equivalency added: ```wavelength.to_value('angstrom', equivalencies=u.spectral())```